ingest/downloader/flattener.py

```diff
@@ -73,8 +74,11 @@ def embed_link_columns(self, entity: Entity):
 
     def _embed_process(self, entity: Entity, embedded_content):
         embed_process = {
-            'process': entity.process.content
+            'process': {
+                'uuid': entity.process.uuid
+            }
         }
+        embed_process['process'].update(entity.process.content)
```

Comment:
Why not set this in the initial dict assignment?

Reply:
how?